tests/by-util/test_cp.rs

```diff
@@ -1626,3 +1626,26 @@ fn test_cp_overriding_arguments() {
         s.fixtures.remove("file2");
     }
 }
+
+#[test]
+fn test_cp_same_file_backup() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    at.touch("source.txt");
+    scene
+        .ucmd()
+        .arg("--backup=simple")
+        .arg("--suffix=.b")
+        .arg("source.txt")
+        .arg("source.txt")
+        .succeeds();
+    assert!(at.file_exists("source.txt.b"));
```

Comment:
Hmm on GNU cp v8.30, I see the following:
```sh
$ touch source.txt && cp --backup=simple --suffix=.b source.txt source.txt
cp: 'source.txt' and 'source.txt' are the same file
```
The command exits with status 1 and `source.txt.b` does not get created. Am I missing something here? I notice that the GNU test case in `tests/cp/backup-1.sh` has a `--force` flag that is not present here.

Reply:
i had the --force but it was failing on Windows (not implemented). 
I will put it back